.github/workflows/coverage.yml

```diff
@@ -15,9 +15,10 @@ jobs:
   coverageReport:
     strategy:
       matrix:
-        api-level: [21,30]
+        api-level: [33]
+        target: [google_apis]
       fail-fast: false
-    runs-on: macos-10.15
+    runs-on: ubuntu-latest
```

Comment:
We should use `macOS`. `ubuntu` os is flaky for UI testing.

Reply:
hi @gouri-panda  this PR is still in draft.